variables.tf

```
@@ -276,17 +287,15 @@ variable "nonpersistent_volume_configs" {
 # Networking
 variable "vpc_id" {
   type        = string
-  description = "Optional; The VPC Id in which resources will be provisioned. Default is the default AWS vpc."
-  default     = ""
+  description = "Required; A vpc-id"
```

I would stick with the naming convention you used in security_group_ids.
```suggestion
  description = "Required; A VPC ID"
```

---

Resolving for now in favor of larger cleanup effort: [IA-4470](https://us-sba.atlassian.net/browse/IA-4470)